### DISCUSSION TITLE
这是0篇测试博文 | Tianzhi Jia (贾添植)

### DISCUSSION THREAD

**Discussion by giscus** [kuizuo/blog#275] [General]
# 这是0篇测试博文 | Tianzhi Jia (贾添植)

作者是一位从来没用过苹果产品的程序员，但在使用了一周的 MacBook Pro 14 寸后，便爱不释手。

https://jiatianzhi.xyz/blog/test-blog0

**Comment by jiatianzhi** [comment]
Good!
